spec/spec_helper.rb

```diff
@@ -1,8 +1,9 @@
 require 'simplecov'
 require 'coveralls'
-require './lib/iosgen'
+require 'thor'
+
+Dir['./lib/**/*.rb'].each { |f| require f }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.